samples/core/src/test/java/io/oasp/gastronomy/restaurant/offermanagement/batch/impl/productimport/OfferManagementTest.java

```diff
@@ -1,5 +1,9 @@
 package io.oasp.gastronomy.restaurant.offermanagement.batch.impl.productimport;
 
+import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.ID_OFFER_SCHNITZEL;

```

Comment:
We should avoid static imports. Why not just qualify the constant where it is used? This makes the code also more readable. Managing and organizing static imports is pretty bad. Hence static imports are more an anti-pattern.